app/templates/frameworks/edit_declaration_section.html

```diff
@@ -6,28 +6,29 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {{ govukBreadcrumbs({
+    "items": [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
       },
       {
-        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-        "label": "Apply to " + framework.name
+        "text": "Apply to " + framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_supplier_declaration_overview", framework_slug=framework.slug),
-        "label": "Your declaration"
+        "text": "Your declaration overview",
+        "href": url_for(".framework_supplier_declaration_overview", framework_slug=framework.slug)
+      },
+      {
+        "text": section.name
```

Comment:
I've just spotted another issue unfortunately, when looking at a DOS4 outcomes service submission

i.e. http://localhost/suppliers/frameworks/digital-outcomes-and-specialists-4/submissions/digital-outcomes/150182

![image](https://user-images.githubusercontent.com/503614/71190166-f5e57980-227b-11ea-910d-631d4c9551ba.png)